2.0/s2i/bin/run

```diff
@@ -1,5 +1,26 @@
 #!/bin/bash
 
+function export_memory_limit() {
+  if [ -f /sys/fs/cgroup/memory/memory.limit_in_bytes ]; then
+    export MEMORY_LIMIT_IN_BYTES=$(cat /sys/fs/cgroup/memory/memory.limit_in_bytes)
+    return 0
+  fi
+  return 1
+}
+
+function export_number_of_cores() {
+  if [ -f /sys/fs/cgroup/cpuset/cpuset.cpus ]; then
+    export NUMBER_OF_CORES=`awk ' \
+      BEGIN { i = 0; FS="-"; RS="[,\n]" } \
+      /^[0-9]+-[0-9]+$/ { i += $2-$1+1 }  \
+      /^[0-9]+$/        { ++i }           \
+      END { print i} \
+      ' /sys/fs/cgroup/cpuset/cpuset.cpus`
+    return 0
+  fi
+  return 1
+}
```

Comment:
@mnagy after thinking some more about this, I think it would make sense to do the parsing of all limits in python script, something like:

```
export $(python read_limits.py)
```

The `read_limits.py` will return env var key/pairs with all properties... we can include it in sti-base.